projects/skia/build.sh

```diff
@@ -16,8 +16,10 @@
 ################################################################################
 
 export CXXFLAGS_ARR=`echo $CXXFLAGS | sed -e "s/\s/\",\"/g"`
-$SRC/depot_tools/gn gen out/Fuzz --args='cc="'$CC'" cxx="'$CXX'" is_debug=false extra_cflags=["'"$CXXFLAGS_ARR"'","-DIS_FUZZING"] skia_use_system_freetype2=false skia_use_fontconfig=false skia_enable_gpu=false extra_ldflags=["-L'$LIB_FUZZING_ENGINE'", "'"$CXXFLAGS_ARR"'"]'
+$SRC/depot_tools/gn gen out/Fuzz --args='cc="'$CC'" cxx="'$CXX'" is_debug=false extra_cflags=["'"$CXXFLAGS_ARR"'","-DIS_FUZZING"] skia_use_system_freetype2=false skia_use_fontconfig=false skia_enable_gpu=false extra_ldflags=["-lFuzzingEngine", "'"$CXXFLAGS_ARR"'"]'
 
 
 
-$SRC/depot_tools/ninja -C out/Fuzz fuzz
\ No newline at end of file
+$SRC/depot_tools/ninja -C out/Fuzz fuzz_region_deserialize
+
+cp out/Fuzz/fuzz_region_deserialize $OUT/fuzz_region_deserialize
```

Comment:
Can you remove the fuzz_ from the fuzz target prefix. We need .options file and fuzz target to have same name.